packages/blockchain-wallet-v4-frontend/src/data/preferences/reducers.js

```diff
@@ -46,6 +47,9 @@ const preferences = (state = INITIAL_STATE, action) => {
       const { displayed } = payload
       return assoc('showBitcoinCashWelcome', displayed, state)
     }
+    case AT.HIDE_KYC_COMPLETED: {
+      return assoc('showKycCompleted', false, state)
```

Comment:
I think we want to hide the KYC notification permanently once the user successfully completes it. Does this do that?

Reply:
Yes, as soon as the action is called showKycCompleted becomes false and then this is stored along with all other preferences in localStorage via reduxPersistStore

Reply:
Nice!